Vagrantfile

```diff
@@ -1,15 +1,23 @@
+# -*- mode: ruby -*-
+
 Vagrant.configure("2") do |config|
   config.vm.box = "centos/7"
+
   config.vm.provider "virtualbox" do |vb|
-  config.vm.network "forwarded_port", guest: 80, host: 8080, auto_correct: true
-  config.vm.network "forwarded_port", guest: 443, host: 8443, auto_correct: true
-  config.vm.network "forwarded_port", guest: 4063, host: 4063, auto_correct: true
-  config.vm.network "forwarded_port", guest: 4064, host: 4064, auto_correct: true
-    vb.customize ["modifyvm", :id, "--memory", "2048"]
+    config.vm.network "forwarded_port", guest: 80, host: 8080, auto_correct: true
+    config.vm.network "forwarded_port", guest: 443, host: 8443, auto_correct: true
+    config.vm.network "forwarded_port", guest: 4063, host: 4063, auto_correct: true
+    config.vm.network "forwarded_port", guest: 4064, host: 4064, auto_correct: true
   end
 
-  config.vm.provision "ansible" do |ansible|
-    ansible.playbook = "playbook.yml"
-    ansible.galaxy_role_file = "../requirements.yml"
+  config.vm.provider "virtualbox" do |vb|
+    vb.memory = "2048"
+    vb.cpus = "2"
   end
+
+  # Uncomment this to automatically run Ansible
+  # config.vm.provision "ansible" do |ansible|
+  #   ansible.playbook = "playbook.yml"
```

Comment:
Would you mind making this callable via the command-line? https://www.vagrantup.com/docs/provisioning/basic_usage.html#run-once-always-or-never

Reply:
I've tried
```
  # Run the Ansible provisioner after the VM has started:
  # vagrant provision --provision-with ansible
  config.vm.provision "ansible", run: "never" do |ansible|
    ansible.playbook = "playbook.yml"
    ansible.galaxy_role_file = "requirements.yml"
  end
```
but I can't get it to run, neither `vagrant provision --provision-with ansible` nor `vagrant provision` work

Reply:
Alright. Probably not worth any more time investment. In general, I find "uncomment this block" unfortunate from a usability point-of-view, but better than nothing.